### PR TITLE
[Rule Tuning] Guided Onboarding Rule

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

Resolves https://github.com/elastic/ia-trade-team/issues/247

## Summary

Reduces the amount of data exposed to the rule to avoid performance issues.

### CODE REVIEW

File: rules/cross-platform/guided_onboarding_sample_rule.toml [hunk 1]
```diff
@@ -15,18 +15,8 @@ enabled = false
...
-index = [
-    "apm-*-transaction*",
-    "auditbeat-*",
-    "endgame-*",
-    "filebeat-*",
-    "logs-*",
-    "packetbeat-*",
-    "traces-apm*",
-    "winlogbeat-*",
-    "-*elastic-cloud-logs-*",
```
Comment (w0rk3r) [inline rules/cross-platform/guided_onboarding_sample_rule.toml:28]: It needs setup, not default. Also, the user is not recommended to ship those logs to the same instance.

image

File: rules/cross-platform/guided_onboarding_sample_rule.toml [hunk 2]
```diff
@@ -15,18 +15,8 @@ enabled = false
...
-]
+from = "now-1h"
```
Comment (w0rk3r) [inline rules/cross-platform/guided_onboarding_sample_rule.toml:18]: It works, but it is strange

image

Comment (terrancedejesus) [inline rules/cross-platform/guided_onboarding_sample_rule.toml:18]: @approksiu does 1 hour make sense for this, or can we reduce it more to maybe last 30 mins? If this is specifically for guided onboarding I would assume 30m would be sufficient for data ingestion when onboarding?

Comment (w0rk3r) [reply rules/cross-platform/guided_onboarding_sample_rule.toml]: Done

**Review by terrancedejesus — APPROVED** [review]
Good work, thank you @w0rk3r!

**Comment by Aegrah** [comment]
Should we bump the updated_date? 

And it says
```
min_stack_comments = "Guided Onboarding will be available in Elastic 8.6+"
min_stack_version = "8.7.0"
```

Should it be 8.7+? Or should min_stack be 8.6.0?

**Review by Aegrah — APPROVED** [review]
As long as we considered the min_stack, it's fine with me. LGTM!

**Pull request merged**